### PR TITLE
Visual tweaks to some screens

### PR DESCRIPTION
**Pull request by pzapataf**

### CODE REVIEW

**Comment by vblagoje** [comment]
I'll take this one @tristantarrant I'll collapse them all in one commit @pzapataf before I integrate them into master.

**Comment by vblagoje** [comment]
@pzapataf looks good but I noticed that "Actions" pull down menus now require two clicks to get activated. I tried both Chrome and Firefox. Would you please confirm what I experienced and if so - please investigate why is that the case! On the current master, I can not notice this issue!

**Comment by vblagoje** [comment]
@pzapataf I found the culprit - it is dropdown.js introduced in these commits When I removed it I got the the desired menu drop-down behaviour. Do we really need dropdown.js?

File: src/main/webapp/cluster-view/cluster-view.html [hunk 1]
```diff
@@ -17,11 +17,6 @@
           <span class="caret"></span>
         </button>
         <ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu">
-          <li role="presentation" class="disabled"><a id="dropdownMenuStart" role="menuitem">Start</a></li>
-          <li role="presentation" class="disabled"><a id="dropdownMenuStop" role="menuitem">Stop</a></li>
-          <li role="presentation" class="divider"></li>
-          <li role="presentation" class="disabled"><a id="dropdownMenuDelete" role="menuitem">Delete</a>
-          <li role="presentation" class="divider"></li>
           <li role="presentation" ng-class="currentCluster.rebalancingStatus ? 'disabled' : 'enabled'"><a  id="dropdownMenuEnableRebalance"  role="menuitem" ng-click="confirmAndSetCacheContainerRebalance(true, 'ENABLE rebalancing for cache container?');">Enable rebalancing</a>
```
Comment (vblagoje) [inline src/main/webapp/cluster-view/cluster-view.html]: I believe the HTML structure we want to here is 
````
<li><a></a></li>
`````

File: src/main/webapp/cluster-view/tasks-view.html [hunk 2]
```diff
@@ -16,24 +16,53 @@
           Actions
           <span class="caret"></span>
         </button>
-        <ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu1">
-          <li role="presentation" class="disabled"><a role="menuitem">Start</a></li>
-          <li role="presentation" class="disabled"><a role="menuitem">Stop</a></li>
+        <ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu">
```
Comment (vblagoje) [inline src/main/webapp/cluster-view/tasks-view.html]: Here as well:
````
<li><a></a></li>
`````

**Comment by pzapataf** [comment]
Fixed the issue, removed the dropdown.js dependency and changed the style of the logout menu. However, a small issue persists when the resolution is collapsed to minimum size, that menu is not expanded on click.

**Comment by vblagoje** [comment]
I think that is not a big issue at this time @pzapataf We can tackle it later. I am going to squash these commits now and integrate it to master!

**Comment by vblagoje** [comment]
Integrated with 6de28b371e2632ff56f2150f146e5757a9d98a3d Thanks @pzapataf

**Pull request closed without merging**